src/OrbitVulkanLayer/SubmissionTracker.h

```diff
@@ -221,7 +221,12 @@ class SubmissionTracker : public VulkanLayerProducer::CaptureStatusListener {
     // other than the debug markers then.
     {
       if (command_buffer_to_state_.contains(command_buffer)) {
-        ERROR("Calling vkBeginCommandBuffer on a command buffer that is not in the initial state.");
+        if (!logged_begin_when_not_initial_) {
```

Comment:
This (and below) seems to be out-of-scope for the change? At least according to the pull request description.